app/services/waste_exemptions_engine/refresh_companies_house_name_service.rb

```diff
@@ -7,6 +7,8 @@ class RefreshCompaniesHouseNameService < WasteExemptionsEngine::BaseService
     def run(company_reference)
       registration = Registration.find_by(reference: company_reference)
       company_name = DefraRubyCompaniesHouse.new(registration.company_no).company_name
```

Comment:
What type of exception does RestClient raise if the URL is invalid or the request times out? Need to handle that as well.